### PR TITLE
Feat Welcome

### PR DESCRIPTION
**Pull request by sa8b**
1. Added content to welcome page

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -47,18 +51,15 @@ export default function Home() {
```
Comment (elquespera) [inline pages/index.tsx:51]: Maybe it's better to wrap it in a div and use max width for large screens? It looks rather wide here:

image

File: pages/index.tsx [hunk 2]
```diff
@@ -67,6 +68,52 @@ export default function Home() {
...
+            <li>CSS3 / HTML5</li>
+            <li>Chrome DevTools, Figma</li>
+            <li>Understanding of the REST</li>
+          </ul>
+          <a
+            href="https://wearecommunity.io/events/rs-react-2023q1"
+            className="px-5 py-2 bg-pink-600 inline-block my-6 mx-auto hover:bg-pink-500 text-white"
+          >
+            Enroll
+          </a>
```
Comment (elquespera) [inline pages/index.tsx:117]: Please make it look more like our other buttons/links with rounded corners and similar padding

**Review by elquespera — CHANGES_REQUESTED** [review]
Nice job!

**Comment by sa8b** [comment]
Fixed all requests

**Comment by Tomas-Zagorskis** [comment]
I think you forgot to commit your changes O.o

**Review by elquespera — APPROVED** [review]
Much better!

**Review by Tomas-Zagorskis — APPROVED** [review]

**Pull request merged**